src/app/_services/api/user-stats.service.ts

```diff
@@ -24,6 +24,12 @@ export class UserStatsService {
             .pipe(catchError(this.handleError<UserStats>(`getUserStat`)));
     }
 
+    getAllUserStat(): Observable<UserStats[]> {
```

Comment:
Are you using this? Should be removed if you are not.

Reply:
Removed.